aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/BaseHandlerStd.java

```diff
@@ -304,53 +294,12 @@ private ProgressEvent<ResourceModel, CallbackContext> modifyParameters(final Pro
         return ProgressEvent.progress(model, context);
     }
 
-    protected boolean isDBClustersAvailable(final ProxyClient<RdsClient> proxyClient, final ResourceModel model) {
-        return proxyClient
-                .injectCredentialsAndInvokeIterableV2(Translator.describeDbClustersRequest(), proxyClient.client()::describeDBClustersPaginator)
-                .stream()
-                .flatMap(describeDbClustersResponse -> describeDbClustersResponse.dbClusters().stream())
-                .filter(dbCluster -> dbCluster.dbClusterParameterGroup().equalsIgnoreCase(model.getDBClusterParameterGroupName()))
-                .allMatch(dbCluster -> dbCluster.status().equals(AVAILABLE));
-    }
+    protected ProgressEvent<ResourceModel, CallbackContext> waitForStabilization(final ProgressEvent<ResourceModel, CallbackContext> progress) {
 
-    protected ProgressEvent<ResourceModel, CallbackContext> waitForDbClustersStabilization(final ProgressEvent<ResourceModel, CallbackContext> progress,
-                                                                                           final AmazonWebServicesClientProxy proxy,
-                                                                                           final ProxyClient<RdsClient> proxyClient) {
-        return proxy.initiate("rds::stabilize-db-cluster-parameter-group-db-clusters", proxyClient, progress.getResourceModel(), progress.getCallbackContext())
-                .translateToServiceRequest(Function.identity())
-                .makeServiceCall(EMPTY_CALL)
-                .stabilize((request, response, proxyInvocation, model, context) -> withProbing(
-                        context,
-                        "db-cluster-parameter-group-db-clusters-available",
-                        3,
-                        () -> isDBClustersAvailable(proxyInvocation, model)))
-                .handleError((describeDbParameterGroupsRequest, exception, client, resourceModel, ctx) ->
-                        Commons.handleException(
-                                ProgressEvent.progress(resourceModel, ctx),
-                                exception,
-                                DB_CLUSTERS_STABILIZATION_ERROR_RULE_SET))
-                .progress();
+        CallbackContext callbackContext = progress.getCallbackContext();
+        if(!callbackContext.isParameterGroupStabilized())
```

Comment:
You don't update this variable to indicate a successful stabilization. Could this lead to the lambda never getting stuck in a loop of stabilization? Or is this parameter updated implicitly?